### PR TITLE
Improve icons in the Link Collection view

### PR DESCRIPTION
**Pull request by endim8**
improve icons for organicmaps & figuregrounder, use osm icon for openorienteeringmap (better than nothing)

new icons:
exsdfjsbkj

### CODE REVIEW

File: app/src/main/java/de/westnordost/streetcomplete/data/user/achievements/AchievementsModule.kt [hunk 1]
```diff
@@ -411,7 +411,7 @@ private val links = listOf(
         "https://oomap.co.uk/global/",
         "OpenOrienteeringMap",
         LinkCategory.GOODIES,
-        null,
+        R.drawable.ic_link_openstreetmap,
```
Comment (westnordost) [inline app/src/main/java/de/westnordost/streetcomplete/data/user/achievements/AchievementsModule.kt:414]:
```suggestion
        R.drawable.ic_link_openorienteeringmap,
```

**Comment by westnordost** [comment]
Hmm to be honest, I like the current figuregrounder-icon better. But I wouldn't mind if that current icon was round instead of a square.

I also noticed that openorienteeringmap has a favicon - I created an SVG of it and added it.

Regarding organic maps, see @FloEdelmann - I think it is better to show 1:1 the app icon

**Comment by matkoniecz** [comment]
> Hmm to be honest, I like the current figuregrounder-icon better.

The same, new one looks like a blob. Have they changed icon?

**Comment by westnordost** [comment]
I'll close this then

**Pull request closed without merging**